book/cli/help.rs

```diff
@@ -0,0 +1,311 @@
+#!/usr/bin/env cargo +nightly -Zscript
```

Comment:
Needs `-S` passed to `env` for multiple arguments

Reply:
Added